quadrotor_propulsion:
        Psi: 0.007539037943
        J_M: 1.783773257e-05
        R_A: 0.4605412456
        k_t: 0.01534796772

        CT2s: -1.3077e-2 
        CT1s: -2.5224e-4 
        CT0s:  1.5278e-5 

        l_m: 0.275

